src/lib/modules/Patron/PatronCurrentLoans/actions.test.js

```diff
@@ -46,7 +46,7 @@ describe('Patron current loans tests', () => {
 
       store.dispatch(actions.fetchPatronCurrentLoans(2)).then(() => {
         expect(mockFetchPatronCurrentLoans).toHaveBeenCalledWith(
-          '(patron_pid:2 AND state:(ITEM_ON_LOAN))&sort=-mostrecent&page=1'
+          '(patron_pid:2 AND state:(ITEM_AT_DESK OR ITEM_ON_LOAN OR ITEM_IN_TRANSIT_FOR_PICKUP OR ITEM_IN_TRANSIT_TO_HOUSE))&sort=-mostrecent&size=15&page=1'
```

Comment:
ping @ntarocco @zzacharo too, these additional states were leftovers or they are left on purpose ? we don't have then in the backend config.